### DISCUSSION TITLE
Fabric  Simple Voice Chat not working

### DISCUSSION THREAD

**Discussion by Goten87** [itzg/docker-minecraft-server#1422] [Q&A] [answered]
hello all im testing and trying to setup a fabric server with the mod Simple Voice Chat
i ported the port as well but my client dont seem to want to connect to it its keep screaming at me
```[VoiceChatAuthenticationThread/INFO]: Trying to authenticate voice connection```
anyone know how to make this work i added the port the mod needs to my docker-compose file but it will not connect for me
(i dont have a firewall up on my test server)

**Comment by Goten87** [comment, marked as answer]
Fixed this issue already 

change the port to `24454:24454/udp` that will work

**Reply by MaxLevs** [reply]
changed port where?

**Reply by Goten87** [reply]
> changed port where?

in the docker compose file

**Reply by BerryGodJJ** [reply]
What software are you using to host your server? I'm using AMP since I have a second PC that I have my servers running off of. I've been also having issues trying to set up the voice chat and I still haven't been able to to figure it out and it's been stressful T0T
